arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Sequence.kt

```diff
@@ -595,7 +682,7 @@ public fun <A> Sequence<A>.salign(
 /**
  * aligns two structures and combine them with the given [Semigroup.combine]
  */
-@Deprecated(SemigroupDeprecation, ReplaceWith("salign(other, SG::combine)", "arrow.typeclasses.combine"))
+@Deprecated(SemigroupDeprecation, ReplaceWith("salign(other, {a, b -> a + b})", "arrow.typeclasses.combine"))
```

Comment:
Is this correct if `SG` is not in scope?

Reply:
We talked like a couple of weeks a go, that over the 90-ish % of the cases will work with the replacement of `combine` with the `+`. For the rest, I'll add an entry to the migration guide indicating this matter.